manifests/bosh-manifest/deployments/010-bosh-template.yml

```diff
@@ -52,8 +52,6 @@ jobs:
       password: (( grab secrets.bosh_nats_password ))
 
     redis:
-      listen_address: 127.0.0.1
-      address: 127.0.0.1
```

Comment:
The same than in https://github.com/alphagov/paas-cf/pull/463/commits/f6afdf6925fbaa055f10a163389881dc717a0d5c#r77513103, being explicit is also good. It would help you to easily know where is the redis server running, instead of having to go to the spec to learn the default.